tests/registry/test_redis_registry.py

```diff
@@ -196,8 +196,8 @@ def test_multiple_save(self):
         self.registry.save_multiple(
             skeys=self.skeys,
             dict_artifacts={
-                "AE": KeyedArtifact(dkeys=["AE"], artifact=VanillaAE(10)),
-                "scaler": KeyedArtifact(dkeys=["scaler"], artifact=StandardScaler()),
+                "AE": KeyedArtifact(dkeys=["AE"], artifact=VanillaAE(10), stateful=True),
```

Comment:
do we need to provide this everytime?

Reply:
What do you think about it? The other option is set a default value for it

Reply:
This should inherently be assigned.

Reply:
For redis registry it is not used anywhere, so I am thinking to make it optional value.